.github/workflows/check-links.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           url: https://docs.openziti.io/
           cmd_params: >
             --color=always 
-            --exclude=https://(EXWPKK5PV4-dsn.algolia.net|www.google.com/search|www.googletagmanager.com|mermaid-js.github.io|mermaid.live|mermaid.ink|play.google.com/|apps.apple.com/|www.reddit.com/r/openziti|github.com/.*/releases/latest/download/)|(#docusaurus_skipToContent_fallback|\.(ziti|svc)(/[^.]+)?)$
+            --exclude=https://(EXWPKK5PV4-dsn.algolia.net|www.google.com/search|www.googletagmanager.com|mermaid-js.github.io|mermaid.live|mermaid.ink|play.google.com/|apps.apple.com/|www.reddit.com/r/openziti|github.com/.*/releases/latest/download/)|(#docusaurus_skipToContent_fallback|\.(ziti(k8s)?|svc)(/[^.]+)?)$
```

Comment:
this is almost comically long already. it really makes me think there must be a way to "do this differently"? :)  maybe a "only check links that link to THIS site instead? like an --include?